### ISSUE TITLE
Graphics_oM: appropriately rename `IRender` interface

### ISSUE THREAD

**Issue by alelom** [BHoM/BHoM#1272]
#### Broken rules:

`IRender` is too generic. There is no way of understanding from the name itself that it actually only asks for a single property, `Colour`, to be implemented.

This prevents future composability. 

#### Suggestions to restore compliance:

We should rename it to something like `IColourable`, or `IColour`, `IColoured`.

**Comment by alelom** [comment]
@al-fisher I think I'd go for `IColourable` to respect the general interface naming convention. What do you think?

**Comment by al-fisher** [comment]
Yep - would agree with following convention - `IColourable` works.
